src/main/java/com/toofifty/easyblastfurnace/methods/GoldBarMethod.java

```diff
@@ -63,12 +63,23 @@ public MethodStep next(BlastFurnaceState state)
             return waitForBars;
         }
 
-        if (state.getFurnace().has(ItemID.GOLD_BAR)) {
-            if (!state.getEquipment().hasIceGlovesEffect()) {
-                return equipIceOrSmithsGloves;
-            }
-            return collectBars;
-        }
+		if (state.getConfig().tickPerfectMethod()) {
```

Comment:
Because of changes to this method and to GoldHybridMethod and MetalBarMethod, tests are failing. Please take a look at the failing tests and update them as necessary.